src/mono/mono/mini/CMakeLists.txt

```diff
@@ -251,6 +252,8 @@ set(llvm_sources
     mini-llvm.c
     mini-llvm-cpp.cpp
     llvm-jit.cpp)
+else()
+set(llvm_sources)
```

Comment:
Nit: indentation (here and below)

Reply:
mono doesn't have a convention for cmake files, so this uses the normal cmake convention of not putting spaces after (.